tools/xray_flux_forecast/v4_find_ys.py

```diff
@@ -24,8 +24,8 @@
 import os
 import os.path
 
-filePath = "dataset_models/sdo/"
-dependent_variable_path = "dataset_models/sdo/y/"
+filePath = "/home/amunozj/FlareAIA/"
```

Comment:
These should stay as relative paths.

Reply:
does this mean that we need to put the data inside the folder structure?  that is not possible in my case

Reply:
Maybe we need to have this file reference the global config.yml?

Reply:
Then you could change the config once and not commit the changes.

Reply:
that's the way it was before, I believe.

Reply:
Click through to this comment and you'll see how it was before.